public/api/private/config

```diff
@@ -36,12 +36,13 @@
     "imprint": "https://example.com/imprint"
   },
   "version": {
-    "version": "mock",
-    "sourceCodeUrl": "https://www.youtube.com/watch?v=dQw4w9WgXcQ",
-    "issueTrackerUrl": "https://www.youtube.com/watch?v=dQw4w9WgXcQ"
```

Comment:
Why did you remove these?

Reply:
The backend route does not provide those and I think the whole thing can be removed (the version modal needs changes for that), because this was built under the assumption that frontend and backend wouldn't be delivered together. As this is now the case this does not make sense anymore…

Reply:
So we hardcode the urls now?.. or put them into the version json?

Reply:
I'm not to sure… I'd suggest removing the two columns and just have the version and the link to the code repo and the issue tracker. Those should the hardcoded then…

Maybe it would be a good idea to add a issue template about GUI issues or something similiar in the hedgedoc/hedgedoc repo which link to an issue in the react-client?

Reply:
Please don't hard code them. Put them at least into a json file. Or read read them from the package.json.

Reply:
Seems like a reasonable way of doing things…

Reply:
Reminder that we've got a links.json file...

Reply:
I reworked this. Have a look